### ISSUE TITLE
An import path cannot end with a '.ts' extension when using workspace version of TypeScript

### ISSUE THREAD

**Issue by David-Else** [axetroy/vscode-deno#78]
I am still getting the above error, I had it on the last extension version too but thought the new Deno might fix it. I have tried restarting TS and Deno language servers, and loading VS Code again.

See the screenshots. I am using the Workspace TS, not the built in VS Code version

**Screenshots**
![Screenshot from 2020-02-29 15-41-00](https://user-images.githubusercontent.com/12832280/75610579-7214fc80-5b0a-11ea-825a-43ecab47c7ff.png)
![Screenshot from 2020-02-29 15-40-51](https://user-images.githubusercontent.com/12832280/75610581-73462980-5b0a-11ea-829d-d1b8b2b0b6a1.png)

deno: 0.35.0
extension: 3

**Comment by axetroy** [comment]
https://github.com/axetroy/vscode-deno/blob/d61253078e25c7edca06bfa08112453ac335f118/package.json#L40-L41

currently, Extension will not allow use Typescript in the workspace

I am not sure if it is caused by this

**Comment by David-Else** [comment]
I want to use the same VS Code TypeScript with intellisense and features available as the project uses so they are consistent. In this case TypeScript 3.7 has some serious bugs that made me use TS 3.8.

It is an important feature, I think the extension should support it, or if that is impossible then say so very clearly with a warning. At the moment it gives no warning and allows `"enableForWorkspaceTypeScriptVersions": false ` to be true... or it seems to.

I found more details on this problem:
- Install a project that has TS as a dependency
- Display a TS file, and go to VS Code and select the version number of TS in the VS Code GUI, click it, click use workspace version.
- This is created in settings.json:
 `"typescript.tsdk": "node_modules/typescript/lib"`
- Now you get extension error
- Use GUI to go back and select workspace TS
- you STILL get the extension error even though you have gone back to worspace version
- look in settings.json and `"typescript.tsdk": "node_modules/typescript/lib"` is STILL there
- delete it and error SOMETIMES goes away, sometimes not, there is some problem here.

basically there is something wrong with toggling to use workspace version of TS, it does not delete the `"typescript.tsdk": "node_modules/typescript/lib"`  and the deno extension does not update and work.

Cheers, keep up the great work!

**Comment by axetroy** [comment]
I am update for this in v3.0.1. It works on my macOS.

If it doesn't work for you, then re-open issue.

**Comment by piotrpalek** [comment]
I'm getting this issue, just installed the extension and have deno in my PATH on Windows 10.
